core/scripts/ocr2dr/templates/oracle.toml

```diff
@@ -0,0 +1,31 @@
+type = "offchainreporting2"
+schemaVersion = 1
+name = "OCR2DR"
+forwardingAllowed = false
+maxTaskDuration = "30s"
+contractID = "{{oracle_contract_address}}"
+ocrKeyBundleID = "{{ocr2_key_bundle_id}}"
+p2pv2Bootstrappers = [
+  "{{p2p_bootstrapper}}"
+]
+relay = "evm"
+pluginType = "directrequest"
+transmitterID = "{{node_eth_address}}"
+observationSource = """
+    decode_log   [type="ethabidecodelog" abi="OracleRequest(bytes32 requestId, bytes data)" data="$(jobRun.logData)" topics="$(jobRun.logTopics)"]
```

Comment:
Update to  OracleRequest(bytes32 indexed requestId, uint64 subscriptionId, bytes data)